JiraReporter/DependencyInjection.cs

```diff
@@ -42,7 +42,7 @@ public static void RegisterServicesFromAssembly<TAssemblyType>(this ContainerBui
 
             builder.RegisterTypes(types)
                 .AsImplementedInterfaces()
-                .PropertiesAutowired()
+                .PropertiesAutowired(PropertyWiringOptions.AllowCircularDependencies)
```

Comment:
Where is this needed? We should avoid circular dependencies

Reply:
I tried generating a reporting and, because of what I have developed, that can't be done anymore without allowing circular dependencies

Reply:
TaskSchedulerService -> DataService -> SettingsService -> TaskSchedulerService